### PR TITLE
[RUNTIME][WEB] Cleanup logging for web runtime.

### PR DESCRIPTION
**Pull request by tqchen**
The log(info) stops working for web runtime due to the usage of time
function.

- Introduce TVM_LOG_CUSTOMIZE anf TVM_LOG_STACK_TRACE.
- Reorganize the log customization code to wasm_runtime(so non-gpu usecase can apply).
- Update the testcase to cover the logging.

### CODE REVIEW

**Comment by tqchen** [comment]
cc @tkonolige @junrushao1994

File: include/tvm/runtime/logging.h [hunk 1]
```diff
@@ -55,6 +66,15 @@
 #define TVM_USE_LIBBACKTRACE 0
 #endif
 
+/*!
+ * \brief Whether or not customize the logging output.
+ *  If log customize is enabled, the user must implement
+ *  runtime::detail::LogFatalImpl and runtime::detail::LogMessageImpl
```
Comment (tkonolige) [inline include/tvm/runtime/logging.h:72]:
```suggestion
 *  tvm::runtime::detail::LogFatalImpl and tvm::runtime::detail::LogMessageImpl
```

**Review by tkonolige — APPROVED** [review]
Thanks Tianqi!

**Comment by junrushao1994** [comment]
Thanks @tqchen @tkonolige! I am going to merge it in :-)

**Review by junrushao1994 — APPROVED** [review]

**Pull request merged**